docs/first-install.md

```diff
@@ -0,0 +1,81 @@
+# First Install
+
+This is similar to [test-deployment.md](test-deployment.md), however, this document is intended for
+the first deployment onto a server with an existing, older `mod-calendar` running.
```

Comment:
What about these instructions are specific to an upgrade?

I think referring to an upgrade as the first install might be somewhat confusing to folks.

Reply:
I've rearranged the docs to resolve this and combine the installation instructions